service/service.hpp

```diff
@@ -149,6 +149,12 @@ namespace raven
     }
 
     //! Helpers
+    void send_answer(json::json &&response_json_data, uvw::PipeHandle &sock) noexcept
```

Comment:
Why do you take this as an rvalue reference ? I don't think sending an object should require ownership over it

Reply:
It's Outdated